src/mmw/js/src/modeling/views.js

```diff
@@ -66,10 +67,83 @@ var ModelingHeaderView = Marionette.LayoutView.extend({
 // The drop down containing the project name
 // and projects options drop down.
 var ProjectMenuView = Marionette.ItemView.extend({
-    model: models.ProjectModel,
+    ui: {
+        rename: '#rename-project',
+        share: '#share-project',
+        remove: '#delete-project',
+        print: '#print-project'
```

Comment:
I think it can be nice to see in the DOM which elements are bound to events and what they do, which is not always easy. Though in this case it may be because of the ID names.